### PR TITLE
Add exit code when 'diff' returns differences

### PR DESCRIPTION
**Pull request by rparkhunovsky**

### CODE REVIEW

File: lib/cloudformation-ruby-dsl/cfntemplate.rb [hunk 1]
```diff
@@ -246,7 +249,7 @@ def cfn(template)
       puts
     end
 
-    exit(true)
+    tags_diff.empty? && params_diff.empty? && template_diff.empty? ? exit(true) : exit(2)
```
Comment (jonaf) [inline lib/cloudformation-ruby-dsl/cfntemplate.rb]: I much prefer using an `if` or `unless` clause here, for readability, than the ternary operator.

File: lib/cloudformation-ruby-dsl/cfntemplate.rb [hunk 2]
```diff
@@ -177,7 +177,10 @@ def cfn(template)
     # example: <template.rb> diff my-stack-name --parameters "Env=prod" --region eu-west-1
     # Diff the current template for an existing stack with the expansion of this template.
 
-    # We default to "output nothing if no differences are found" to make it easy to use the output of the diff call from within other scripts
+    # `diff` operation exit codes are:
```
Comment (jonaf) [inline lib/cloudformation-ruby-dsl/cfntemplate.rb]: If there's a sensible place for it, I'd like to see this in the documentation for the `diff` subcommand as well.

Comment (rparkhunovsky) [inline lib/cloudformation-ruby-dsl/cfntemplate.rb]: README.md may be the place.

**Comment by jonaf** [comment]
LGTM

**Pull request closed without merging**